CHANGES

```diff
@@ -1,5 +1,15 @@
 # changes since last release
 
+  -- Source terms (gravity, rotation, etc.) have now been all coalesced into
+     a single MultiFab. This reduces the memory footprint of the code. This
+     negates the need for the code parameters update_state_between_sources and
+     coalesce_update_diagnostics, so they have been removed. This will cause
+     a change in results: the previous code behavior was to update the state
+     with each successive source term as it was applied at the new time. Now
+     every source term will be calculated using the same new-time state (the one
+     coming out of the hydro) and the source terms are all applied to the state
+     in one shot at the end.
```

Comment:
perhaps note that both methods are second-order accurate

Reply:
Done